### ISSUE TITLE
read only after imaging to new card (homeassistant)

### ISSUE THREAD

**Issue by CarbonUnderPressure** [Drewsif/PiShrink#303]
I am missing something here, but is it expected to be read-only after shrinking and copying the image to a new SD card? Is there a way to still edit the files?
For reference, I am cloning Home Assistant after removing sensors and devices. Home Assistant seems to be buggy using the shrunk image. I would also like to be able to edit my files in Windows before reuploading them to a new Raspberry Pi 4.

**Comment by Drewsif** [comment]
It should not be read only no. Did you cleanly shutdown before taking the image? I can't say Ive ran into this issue before.

**Comment by Drewsif** [comment]
This is an automated cleanup message. 

This issue is being closed because Home Assistant OS uses an 8-partition layout, which is currently unsupported for PiShrink's auto-expansion (which targets the last partition). Furthermore, the inability to edit files in Windows is due to Windows not natively supporting the `ext4` filesystem.

If you believe this issue was closed in error, please feel free to reopen it with additional details!
